backend/app/routers/order.py

```
@@ -6,69 +6,29 @@
 from app.database import get_db, SessionLocal
 from app.utils import get_current_user
+from app.helpers.orders import (
+    get_product_or_404,
+    get_order_or_404,
+    get_bill_or_404,
+    get_order_items_details,
+    get_user_brand_ids,
+    validate_artisan_order_access,
+    format_order_response,
+    get_artisan_order_items,
+    get_current_user,
```

Ruff (0.8.2)

Redefinition of unused `get_current_user` from line 8

(F811)